docs/demos/filters/min_max_filter.jl

```diff
@@ -0,0 +1,56 @@
+# Topic : Max Min Filters
+# reference paper : https://core.ac.uk/download/pdf/194053536.pdf
```

Comment:
URLs can easily vanish and becomes invalid. I suggest a formal citation format like what I did in #189 so that when the reader is interested, he always has a clue of what he can search for.